### PR TITLE
feat(products): limit number of tiles displayed

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/manager-hub`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          |  MANAGER-4640
| License          | BSD 3-Clause

## Description

- Limit number of tiles displayed

### CODE REVIEW

File: packages/manager/modules/hub/src/components/products/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -28,5 +28,7 @@
   "manager_hub_products_VEEAM_VEEAM_ENTERPRISE": "Veeam Enterpise",
   "manager_hub_products_VPS": "VPS",
   "manager_hub_products_VRACK": "vRacks",
-  "manager_hub_products_XDSL": "xDSL"
+  "manager_hub_products_XDSL": "xDSL",
+  "manager_hub_products_see_less": "Voir moins de service",
```
Comment (antleblanc) [inline packages/manager/modules/hub/src/components/products/translations/Messages_fr_FR.json]: Isn't supposed to be:
```suggestion
  "manager_hub_products_see_less": "Voir moins de services",
```

File: packages/manager/apps/hub/src/dashboard/dashboard.html [hunk 2]
```diff
@@ -62,7 +62,7 @@ <h2 data-translate="manager_hub_dashboard_overview"></h2>
         </div>
         <h2 data-translate="manager_hub_dashboard_services"></h2>
         <hub-products
-            products="$ctrl.services.data"
+            products="$ctrl.products"
```
Comment (radireddy) [inline packages/manager/apps/hub/src/dashboard/dashboard.html]: data-

File: packages/manager/modules/hub/src/components/products/products.html [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
-        class="col-md-6 col-xl-4 mb-2"
-        data-ng-repeat="(product, data) in $ctrl.products track by product"
+        class="col-md-6 mb-2 mb-md-4"
+        data-ng-repeat="product in $ctrl.products | limitTo: ($ctrl.expand ? $ctrl.products.length : $ctrl.DEFAULT_DISPLAYED_TILES) track by product.productType"
     >
         <ovh-manager-hub-tile
-            heading="{{:: 'manager_hub_products_' + product | translate }}"
-            badge-text="{{ data.count }}"
+            heading="{{:: 'manager_hub_products_' + product.productType | translate }}"
+            badge-text="{{ product.count }}"
```
Comment (radireddy) [inline packages/manager/modules/hub/src/components/products/products.html]: may be one-time binding

File: packages/manager/apps/hub/src/dashboard/dashboard.html (hunk 2 above)
Comment (marie-j) [reply packages/manager/apps/hub/src/dashboard/dashboard.html]: This is a custom component so we've decided it is not mandatory

**Pull request merged**